.ci/scripts/run_fb_dlrm_docker.sh

```diff
@@ -0,0 +1,60 @@
+#!/bin/bash -eEx
```

Comment:
rename run_fb_dlrm_docker.sh to run_dlrm_docker.sh

Reply:
Fixed.